client/verta/tests/test_artifacts.py

```diff
@@ -160,10 +160,20 @@ def test_blacklisted_key_error(self, experiment_run, all_values):
             with pytest.raises(ValueError):
                 experiment_run.log_artifact_path(key, artifact)
 
+    @staticmethod
+    def generate_random_data():
+        while True:
+            data = os.urandom(2 ** 16)
+            bytestream = six.BytesIO(data)
+            try:
+                pickle.load(bytestream)
+            except:
+                return data
```

Comment:
So the logic here is that if it happens to be a data type that pickle recognize (e.g float), then we don't want that )? Sounds/looks good to me.

Reply:
yes